src/shared/ui/InputFiled/imageTypeFiled/WrapperImageInput.tsx

```
@@ -1,13 +1,31 @@
+export default function WrapperImageInput({ ...props }: Props) {
+  const { variant, children, isFocused, className, removeImage } = props;
+
+  const removeButton = (variant: TImageVariant) => {
+    return (
+      <button
+        onClick={removeImage}
+        className={cn(
+          'absolute  z-10',
+          variant === 'profile-circle' ? 'top-5 right-5 bg-primary rounded-md' : ' top-2 right-2'
+        )}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)